bedrock/firefox/templates/firefox/hub/home-quantum.html

```diff
@@ -0,0 +1,390 @@
+    <div class="content">
+      <div class="whats-new-content">
+          <h2 class="section-title">{{ _('What’s new?') }}</h2>
+
+          <ul>
+            <li>{{ _('A powerful, new engine that’s built for rapidfire performance.') }}</li>
+            <li>{{ _('Better, faster page loading that uses less computer memory.') }}</li>
+            <li>{{ _('Gorgeous design and smart features for intelligent browsing.') }}</li>
+          </ul>
+
+          <p class="quote">
+            {{ _('“Faster and even more minimalist, Firefox Quantum makes Chrome look old.”') }} —
+            <cite><a href="https://www.digitaltrends.com/web/firefox-quantum-hands-on-review/">{{ _('(Digital Trends)') }}</a></cite>
+          </p>
+      </div>
+      <div class="whats-new-image">
+        {{ high_res_img('firefox/hub/quantum/new-tab.png', {'alt': _('Screenshot of the redesigned Firefox new-tab page.'), 'width': '849', 'height': '540'}) }}
+      </div>
+    </div>
+  </section>
+
+  <section class="page-section key-features-section">
+    <div class="key-feature" data-scroll-tracking="2x Faster">
+      <div class="content">
+        <div class="key-feature-container">
+          <div class="key-feature-content">
+            <h2 class="section-title">{{ _('2x Faster') }}</h2>
+            <p>
+            {% trans url='https://blog.mozilla.org/firefox-de/firefox-quantum-superschnell-und-schont-den-speicher/' if LANG == 'de' else 'https://blog.mozilla.org/firefox/quantum-performance-test/' %}
+              Crazy <a href="{{ url }}">powerful browser engine</a>? Check. Less time waiting around for pages to load?
+              Also, check. Get the best Firefox yet.
+            {% endtrans %}
+            </p>
+          </div>
+          <div class="key-feature-media video">
+            <div class="moz-video-container">
+              <button class="moz-video-button" type="button" aria-controls="faster-video">{{ _('Watch the video') }}</button>
+              <div class="video-container">
+                <video id="faster-video" preload="metadata" controls poster="{{ static('img/firefox/hub/quantum/quantum-vs-chrome-poster.png') }}" data-ga-label="Quantum vs Chrome Video">
+                  <source src="https://assets.mozilla.net/video/firefox-home/quantum-vs-chrome.webm" type="video/webm">
+                  <source src="https://assets.mozilla.net/video/firefox-home/quantum-vs-chrome.ogv" type="video/ogg; codecs='theora, vorbis'">
+                  <source src="https://assets.mozilla.net/video/firefox-home/quantum-vs-chrome.mp4" type="video/mp4">
+                  <a href="https://www.youtube.com/watch?v=YIywpvHewc0">
+                    <img src="{{ static('img/firefox/hub/quantum/quantum-vs-chrome-poster.png') }}" alt="">
+                  </a>
+                </video>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
`<div>` tree! Maybe add some comments after to denote which parent is being closed?